docs/content/releases/release-notes-1.1.0.md

```diff
@@ -11,6 +11,8 @@ docs: "DOCS-713"
 
 ## NGINX Service Mesh Version 1.1.0
 
+29 Jun 2021
```

Comment:
Why not use the full Month? Ditto for the others, but especially this one. Looks like a typo since it's just missing the `e`.